src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -873,7 +873,7 @@ export const selectAssetEquityItemsByFilter = createDeepEqualOutputSelector(
         underlyingAssetIds: lpOpportunity.underlyingAssetIds,
         underlyingAssetRatiosBaseUnit: lpOpportunity.underlyingAssetRatiosBaseUnit,
         cryptoAmountBaseUnit: lpOpportunity.cryptoAmountBaseUnit,
-        assetId,
+        assetId: lpOpportunity.id,
```

Comment:
Note `id` isn't guaranteed to be an AssetId, though for LP it actually is so that's fine, but `lpOpportunity.assetId` is usually safer